xtask/src/release.rs

```diff
@@ -54,14 +54,18 @@ impl ReleaseTask {
             return Err("This version is already released".into());
         }
 
-        let _dir = pushd(self.get_path())?;
+        let _dir = pushd(self.crate_path())?;
 
-        let remote = Self::get_remote()?;
+        let remote = Self::git_remote()?;
 
-        // println!("Checking status of git repository…");
-        // if !cmd!("git status -s -uno").read()?.is_empty() {
-        //     return Err("This git repository contains untracked files".into());
-        // }
+        println!("Checking status of git repository…");
+        if !cmd!("git status -s -uno").read()?.is_empty() {
+            print!("This git repository contains untracked files.");
+
+            if !Self::ask_continue()? {
```

Comment:
This will ask "skip this step", right? When there isn't actually a step to be skipped.

Reply:
Ah yes…